core/key.go

```diff
@@ -577,6 +625,46 @@ func (k CollectionSchemaVersionKey) ToDS() ds.Key {
 	return ds.NewKey(k.ToString())
 }
 
+func (k SchemaHistoryKey) ToString() string {
+	result := COLLECTION_SCHEMA_VERSION_HISTORY
+
+	if k.SchemaID != "" {
+		result = result + "/" + k.SchemaID
+	}
+
+	if k.PreviousSchemaVersionID != "" {
+		result = result + "/" + k.PreviousSchemaVersionID
+	}
+
+	return result
+}
+
+func (k SchemaHistoryKey) Bytes() []byte {
```

Comment:
suggestion: would be nice to add a test for this like just to improve the test coverage

Reply:
Test coverage is not an end goal.  If anything that this is not covered is a nice indicator that it is dead code and could ultimately be removed.